src/wvtc-reimbursement-races.html

```diff
@@ -35,15 +41,18 @@
       properties: {
         races: {
           type: Object,
-          computed: '_computeRaces(_xcRaces, _roadRaces)',
+          computed: '_computeRaces(_xcRaces, _roadRaces, _mutRaces)',
           notify: true,
           readOnly: true
         },
         year: Number,
         _xcRaces: Object,
-        _roadRaces: Object
+        _roadRaces: Object,
+        _mutRaces: Object,
       },
-      _computeRaces: function(xcRaces, roadRaces) {
+      _computeRaces: function(xcRaces, roadRaces, mutRaces) {
+        // NOTE: not including mutRaces so that the drop down will continue
+        // to work for years prior to 2020 when we didn't include MUT.
```

Comment:
Actually, here if you set for instance `/races/2019/mut` to the empty string then it will work correctly.